widgets/serializers/markdown.py

```diff
@@ -3,13 +3,13 @@
     WidgetConfigSerializer,
     WidgetInstanceSerializer,
 )
-from widgets.serializers.react_fields import ReactCharField
+from widgets.serializers.react_fields import ReactMarkdownWysiwygField
 
 
 class MarkdownWidgetSerializerConfigSerializer(WidgetConfigSerializer):
     """Serializer for MarkdownWidgetSerializer config"""
 
-    source = ReactCharField(help_text="Enter widget text")
+    source = ReactMarkdownWysiwygField(help_text="Enter widget text")
```

Comment:
The help text doesn't display in the source editor.

Reply:
I looked into adding placeholder text to the prosemirror input and it looks nontrivial. Do you mind if we skip it for now and add it as a separate issue?